tests/e2e/tidbcluster/serial.go

```diff
@@ -692,6 +694,32 @@ var _ = ginkgo.Describe("[Serial]", func() {
 				err := utilpod.WaitForPodsAreChanged(c, pods, time.Minute*5)
 				framework.ExpectEqual(err, wait.ErrWaitTimeout, "pods should not change in 5 minutes")
 			})
+
+			ginkgo.Context("Admission Webhook", func() {
+				ginkgo.BeforeEach(func() {
+					setOperatorConfig = func(oa *tests.OperatorConfig) {
+						oa.TestMode = true
+						oa.WebhookEnabled = true
+						oa.StsWebhookEnabled = true
+					}
+				})
+
+				ginkgo.It("should be able to be deployed", func() {
+					ginkgo.By("Wait old webhook pods to become ready")
+					err := oa.WaitAdmissionWebhookReady(ocfg, time.Minute*3, time.Second*10)
+					framework.ExpectNoError(err, "failed to wait old webhook pods to become ready")
+
+					ginkgo.By("Upgrade TiDB Operator and CRDs to current version")
+					ocfg.Tag = cfg.OperatorTag
+					ocfg.Image = cfg.OperatorImage
```

Comment:
Is it needed to check the image after upgrading?

Reply:
Done